docker/Dockerfile.zed1

```diff
@@ -0,0 +1,41 @@
+# Copyright (c) 2023, NVIDIA CORPORATION.  All rights reserved.
+#
+# NVIDIA CORPORATION and its licensors retain all intellectual property
+# and proprietary rights in and to this software, related documentation
+# and any modifications thereto.  Any use, reproduction, disclosure or
+# distribution of this software and related documentation without an express
+# license agreement from NVIDIA CORPORATION is strictly prohibited.
+
+ARG BASE_IMAGE
+FROM ${BASE_IMAGE}
+
+ARG ZED_SDK_MAJOR=4
+ARG ZED_SDK_MINOR=1
+
+# zed-ros2-wrapper dependencies
+RUN --mount=type=cache,target=/var/cache/apt \
+  apt-get update && apt-get install -y \
+  libgeographic-dev  \
+  ros-humble-geographic-info \
+  ros-humble-nmea-msgs \
+  ros-humble-robot-localization \
+  ros-humble-xacro
```

Comment:
We still need to test SVO playback in the container to make sure everything works in this branch

Reply:
svo playback has been tested, and works in RViz